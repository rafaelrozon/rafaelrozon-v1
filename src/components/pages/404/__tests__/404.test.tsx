import * as React from 'react';
import { render } from '../../../../utils/testUtils';
import NotFoundPage from "..";

describe("404 tests", () => {
    it("should render", () => {
        const {container} = render(<NotFoundPage />)
        expect(container).toMatchSnapshot();
    });
});
