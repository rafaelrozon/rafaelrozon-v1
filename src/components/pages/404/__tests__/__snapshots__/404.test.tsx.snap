// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 tests should render 1`] = `
<div>
  <h1
    class="sc-AxjAm jKXhHi"
  >
    NOT FOUND
  </h1>
  <button
    class="sc-AxirZ kPICFO"
  >
    Go Back?
  </button>
  <img
    class="sc-AxiKw juzLaN"
    src="https://media.giphy.com/media/rU6L2fuP17sS4/giphy.gif"
  />
</div>
`;
